draft-ietf-jsonpath-base.md

```diff
@@ -1129,16 +1131,17 @@ return a primitive value or at most one node.
 
 ~~~~ abnf
 comp-expr    = comparable S comp-op S comparable
-comparable   = number / string-literal /        ; primitive ...
-               true / false / null /            ; values only
-               singular-path /                  ; Singular Path value
-               function-expression
+comparable   = number /              ; literal ...
+               string-literal /      ; values ...
+               true / false /        ; must be ...
```

Comment:
I'm not quite sure what this "must" is. Is this an additional requirement? Or is this just the rationale for what the grammar looks like?

Reply:
Rationale. I thought about saying literal values cannot be structured, but that didn't seem appropriate. I'd also be comfortable deleting this comment for readability.

Reply:
Deleted. There is already mention in the text just before the ABNF.